src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    private ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+
+        switch (figureNumber) {
+            case 1:
+                return new Circle(color.getRandomColor(), getLength());
+            case 2:
+                return new Square(color.getRandomColor(), getLength());
+            case 3:
+                return new Rectangle(color.getRandomColor(), getLength(), getLength());
+            case 4:
+                return new RightTriangle(color.getRandomColor(), getLength(),
+                        getLength(), getLength());
+            default:
+                return new IsoscelesTrapezoid(color.getRandomColor(), getLength(),
+                        getLength(), getLength());
+        }
+    }
+
+    int getLength() {
```

Comment:
не забувай за модифікатори
```suggestion
   private int getRandomLength() {
```